frontend/appflowy_flutter/lib/plugins/database_view/widgets/row/cell_builder.dart

```diff
@@ -31,6 +33,32 @@ class GridCellBuilder {
     );
 
     final key = cellContext.key();
+
+    if (PlatformExtension.isMobile) {
+      final mobileGridCellWidget = _getMobileCardCellWidget(
+        cellContext,
+        cellControllerBuilder,
+        style,
+        key,
+      );
+      return mobileGridCellWidget;
+    } else {
+      final desktopGridCellWidget = _getDesktopGridCellWidget(
+        cellContext,
+        cellControllerBuilder,
+        style,
+        key,
+      );
+      return desktopGridCellWidget;
+    }
```

Comment:
Redundant else statement, and variable initializations
```suggestion
    if (PlatformExtension.isMobile) {
      return _getMobileCardCellWidget(
        cellContext,
        cellControllerBuilder,
        style,
        key,
      );
    }
   
    return  _getDesktopGridCellWidget(
      cellContext,
      cellControllerBuilder,
      style,
      key,
    );
```